assets/scripts/script.js

```diff
@@ -50,6 +55,87 @@ function showRandomDrinkSection(){
 // SHOW RANDOM DRINK SECTION
 // button
 // id="randomDrink"
+// ***** FOR NARGIZA'S CODE FOR RANDOM DRINK start
+const getRandomDrinkBtn = document.getElementById('collins');
+const getRandomDrinkContainer = document.getElementById('randomDrinkResult');
+
+getRandomDrinkBtn.addEventListener('click', () => {
+	fetch('https://www.thecocktaildb.com/api/json/v1/1/random.php')
+		.then(res => res.json())
+		.then(res => {
+		createRandomDrink(res.drinks[0]);
+	});
+});
+
+const createRandomDrink = (drinks) => {
+	const ingredients = [];
+	// Get all ingredients from the object. Up to 15
+	for(let i=1; i<=15; i++) {
+		if(drinks[`strIngredient${i}`]) {
+			ingredients.push(`${drinks[`strIngredient${i}`]} - ${drinks[`strMeasure${i}`]}`)
+		} else {
+			// Stop if no more ingredients
+			break;
+        }
+        document.getElementById("homePage").style.display = "block";
```

Comment:
Do we need lines 80 - 85? It looks like they are a duplicate from the function on line 22. 
Just checking to be sure they serve a purpose here.